tests/testthat/test-escape_path.R

```diff
@@ -0,0 +1,27 @@
+context("escape_path")
```

Comment:
My convention is to name the test file the same as the R file - so this should be `test-ignore.R`. Can you please change?